AdyenComponents/BLIK/BLIKComponent.swift

```diff
@@ -9,7 +9,15 @@ import Foundation
 import UIKit
 
 /// A component that provides a form for BLIK payments.
-public final class BLIKComponent: PaymentComponent, PresentableComponent, PaymentAware, LoadingComponent {
+public final class BLIKComponent: PaymentComponent, PresentableComponent, PaymentAware, LoadingComponent, CheckoutBaseCallbacks {
+    
+    package var onSubmit: Adyen.SubmitHandler?
```

Comment:
I would prefer to not see this on top, can we extract this to an extension? I would also play around with the default implementation in the protocol itself if possible to just remove this code from all components and override it if the behavior is different for a specific component.